app/form_models/publishers/job_listing/documents_form.rb

```diff
@@ -38,6 +38,6 @@ def document_presence
     return unless vacancy.include_additional_documents
     return if documents.present?
 
-    errors.add(:documents, :blank) unless vacancy.supporting_documents.any?
```

Comment:
This condition allowed publishers to submit the form without `documents` attached as long as the vacancy already had other `supporting_documents` attached to it. We are removing this possibility